UI/obs-app.cpp

```diff
@@ -2122,6 +2122,11 @@ static int run_program(fstream &logFile, int argc, char *argv[])
 		     rosettaTranslated ? "true" : "false");
 #endif
 
+#if defined(_WIN32)
+		if (IsARM64Emulated())
+			blog(LOG_INFO, "Windows ARM64 Emulation used: true");
```

Comment:
I'm not sold on this wording as it's a bit confusing to someone who doesn't know exactly what it means in advance, but I can't think of something better. Microsoft labels this as "x64 emulation on Arm" and is apparently limited to Windows 11 going forward.

Reply:
How about `IsEmulatedOnARM64Windows()`?

Name aside, the problem is that line of log doesn't show up. I'll try to fix that

Reply:
For the function itself, `IsEmulatedOnARM64()` does sound better. The log message I'm unsure about.